scripts/linters/css_linter.py

```diff
@@ -137,50 +123,43 @@ def _lint_css_files(self):
 
         if num_files_with_errors:
             for result in result_list:
-                python_utils.PRINT(result)
-                summary_messages.append(
+                full_error_messages.append(result)
+                stripped_error_messages.append(
                     self._get_trimmed_error_output(result))
-            summary_message = ('%s %s CSS file' % (
-                linter_utils.FAILED_MESSAGE_PREFIX, num_files_with_errors))
-        else:
-            summary_message = ('%s %s CSS file linted (%.1f secs)' % (
-                linter_utils.SUCCESS_MESSAGE_PREFIX, num_css_files,
-                time.time() - start_time))
-        python_utils.PRINT(summary_message)
-        summary_messages.append(summary_message)
-
-        python_utils.PRINT('CSS linting finished.')
-        return summary_messages
+            failed = True
+
+        return concurrent_task_utils.TaskResult(
+            name, failed, stripped_error_messages, full_error_messages)
```

Comment:
Can we please replicate the same comments of the `codeowner_linter.py` to everywhere else? (Make this self.failed etc.)

Reply:
I've already replicated the codeowner_linter.py changes everywhere except self.failed and self.messages as in codeowner linter we have checks related to each other, while in other linters all checks are independent so we can't do that here.